shelley/chain-and-ledger/executable-spec/test/Test/Shelley/Spec/Ledger/Address.hs

```diff
@@ -81,19 +84,69 @@ goldenTests =
         "addrEnterpriseS"
         putAddr
         (Addr scriptHash StakeRefNull)
-        "7205060708"
+        "7205060708",
+
+      -- serialisation golden tests
+      goldenSerialisation
+        "addrEnterpriseK for network id = 0"
+        (Addr (keyBlake2b224 paymentKey) StakeRefNull)
+        "62",
```

Comment:
that's unexpected result

Reply:
`62` is the correct header for the address you are serializing. the nibble `6` is for an enterprise address with a keyhash. the `2` is the `Offline` network id. The problem is that the key hash is empty, probably because of the `B16.decode hk` below that @redxaxder noticed.

Reply:
thanks -> see my commit https://github.com/input-output-hk/cardano-ledger-specs/pull/1443/commits/8c7a0d69f361f4195edd7ed89203813d7aac3d77